compaction/task_manager_module.hh

```diff
@@ -91,6 +91,26 @@ protected:
     virtual future<> run() override;
 };
 
+class cleanup_compaction_task_impl : public compaction_task_impl {
```

Comment:
For future: it would have been nice to have a sentence about what the purpose of this class is.
Haiving "This will serve as the base class of all concrete cleanup compaction task classes", in the commit message would have saved me a few minutes of review time, searching for where `run()` is implemented.

Reply:
Sure, I'm sorry, I will remember about that.